src/extensibility/ExtensionManager.js

```diff
@@ -260,6 +280,18 @@ define(function (require, exports, module) {
             if (!extensions[id]) {
                 extensions[id] = {};
             }
+            
+            if (locationType === LOCATION_USER) {
+                if (!userExtensions[id]) {
+                    userExtensions[id] = {};
+                }
+                
+                userExtensions[id].installInfo = {
```

Comment:
Can't we also set the data directly on `userExtensions[id]` rather than `userExtensions[id].installInfo`?
Is there a plan to ever have more properties than just `installInfo`?